lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2292,6 +2292,67 @@ definitions:
           - 'NuRAN Cavium OC-LTE'
           - 'FreedomFi One'
         x-nullable: false
+      mme_pool_1:
+        type: string
+        format: ipv4
+        example: ''
+      mme_pool_2:
+        type: string
+        format: ipv4
+        example: ''
+      managementServer:
+        $ref: '#/definitions/managementServer'
+      sync_1588:
+        $ref: '#/definitions/sync_1588'
+  sync_1588:
+    description: sync_1588 Configuration for  eNodeb devices.
+    type: object
+    properties:
+      sync_1588_switch:
+        type: boolean
+        example: false
+      sync_1588_domain:
+        type: integer
+        format: int32
+        example: 0
+      sync_1588_msg_interval:
```

Comment:
include the units, definition (as well as info to look up with a spec), and limits

Reply:
Please also extend this to the other properties in the swagger definitions.